src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.profiles.active=local
```

Comment:
Don't put this here, send this in the command you use to run the application

Reply:
Done.